survos/command-bundle/1.5/config/routes/survos_command.php

```diff
@@ -0,0 +1,12 @@
+<?php
+// config/routes.php
```

Comment:
Remove this command - and this is also more commonly written in YAML, not PHP

Reply:
done